apps/charterafrica/src/payload/fields/ecosystem/airtableBaseSelect.js

```diff
@@ -18,6 +27,11 @@ const getOptions = async () => {
 };
 
 const validateBaseSelect = async (value, { hasMany, required, t }) => {
+  if (value) {
+    const link = getUrl(value);
+    const { tables } = await fetchJson.get(link);
+    schema.tables = tables;
+  }
   const options = await getOptions();
   return select(value, { hasMany, options, required, t });
```

Comment:
```suggestion
  const options = await getOptions();
  const valid = select(value, { hasMany, options, required, t });
  if (valid) {
    const link = getUrl(value);
    const { tables } = await fetchJson.get(link);
    schema.tables = tables;
  }
  return valid;
```

1. We should set schema only when we are sure it's a valid selection
2. Do we need to load bases every time? Can't we cache this too i.e. only load it the first time.